source/frequently-asked-questions/frequently-asked-questions.rst

```diff
@@ -2,3 +2,21 @@
 Frequently Asked Questions
 ################################################################################
 
+* **How can I store big files on the blockchain?**
+  Swarm_ is an Ethereum-specific project for distributed file storage. IPFS_ is an independent project which has close ties to Ethereum; it will be used independently and may be used as the layer underlying Swarm.
```

Comment:
this is not (yet) the case, so misleading without clear qualifiation that it is a future planoid